### ISSUE TITLE
WebGPU 2022-03-29 > 2022-05-15?

### ISSUE THREAD

**Issue by Kangz** [w3c/security-request#23]
In the issue title above add the document name followed by the date of this request, then the date of your proposed deadline for comments.

- **name of spec to be reviewed:** WebGPU and WGSL
- **URL of spec:** http://gpuweb.github.io/gpuweb/ and http://gpuweb.github.io/gpuweb/wgsl

- **Current Rec/Note phase?** FPWD, looking to CR at some point.
- **What and when is your next expected transition?** Transitioning to CR at some point, security considerations are going to change between now and then.
- **What has changed since any previous review?** There was no previous review.
- **Please point to the results of your [self-review](https://w3ctag.github.io/security-questionnaire/)**: https://gpuweb.github.io/gpuweb/explainer/#questionnaire
- **Where and how to file issues arising?** https://github.com/gpuweb/gpuweb/issues
- **Pointer to any explainer for the spec?** https://gpuweb.github.io/gpuweb/explainer/

Other comments: Thanks!

**Comment by Kangz** [comment]
@samuelweiler is there an ETA on security WG reviews? If that helps we could setup a call where some members of the WebGPU WG walk through security aspects of the spec.

**Comment by Kangz** [comment]
@samuelweiler any ETA? This request for review has now been open for almost 3 months. I know WebGPU is a big API, but the group would be happy to help understand key aspects in a call if needed.

**Comment by Kangz** [comment]
Up again! WebGPU is starting to think about going to CR so it would be nice to have the security review done before then.

**Comment by Kangz** [comment]
Up again! Tentatively tagging @samuelweiler but feel free to tag someone else.

This request has now been opened for 5 months. Given the lack of any ETA, if the review comes before WebGPU goes to Candidate Recommendation then that's nice, but if it doesn't, then WebGPU cannot be blocked on the review and will proceed without it.

**Comment by Kangz** [comment]
Up again!
